src/frontend/src/test-e2e/flows.ts

```diff
@@ -178,11 +167,6 @@ export const FLOWS = {
     await addDeviceSuccessView.waitForDisplay();
     await addDeviceSuccessView.continue();
   },
-  skipRecoveryNag: async (browser: WebdriverIO.Browser): Promise<void> => {
```

Comment:
Is there any test left that still goes through the recovery nag? I think there should be one e2e test that does the recovery nag flow. You could trigger that by artificially setting the metadata to some value in the past.

Reply:
No, there is no test that has this flow anymore. I checked a way to advance time, but that's not possible for e2e.

How could I artificially set the metadata to some value in the past?

Shall I expose the function in the console and trigger it? It seems dangerous.

Reply:
Do we do that somewhere else already?

Reply:
No, but we do similar things when we modify alternative origins though.

Reply:
I'm sorry, but I don't find it. Why should the alternative origins create an identity? Where does it do that?

Reply:
As agreed, I tested the flow manually by changing the waiting time to one minute. I was able to test that before one minute the user doesn't see the page and after one minute the page appears. I did it three time to test that updating the metadata also works as expected.

Reply:
Thanks @lmuntaner and as discussed, the e2e test requires some infrastructure so we will do this in a separate PR.